### PR TITLE
New package: notify-osd-0.9.34

### PR DESCRIPTION
**Pull request by LockeAnarchist**

### CODE REVIEW

File: srcpkgs/mdadm/template [hunk 1]
```diff
@@ -1,25 +0,0 @@
-# Template file for 'mdadm'
```
Comment (beefcurtains) [inline srcpkgs/mdadm/template]: Nope nope nope nope nope nope nope

**Comment by Gottox** [comment]
Please remove unneeded commits from this PR. (git fetch https://github.com/voidlinux/void-packages.git;git rebase -i FETCH_HEAD`)

**Comment by Gottox** [comment]
Please add `glib-devel` and `dbus-glib-devel` as hostmakedepends to fix cross build.

File: srcpkgs/notify-osd/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'notify-osd'
+
+pkgname=notify-osd
+version=0.9.34
+revision=1
+build_style=gnu-configure
+short_desc="Canonical's on-screen-display notification agent"
+maintainer="Emanuel Serpa <emanuel@openmailbox.org>"
+hostmakedepends="pkg-config dbus-glib-devel glib-devel"
+makedepends="libwnck-devel libnotify-devel"
```
Comment (xtraeme) [inline srcpkgs/notify-osd/template]: You are missing `dbus-glib-devel` in `makedepends` to fix cross compilation.

File: srcpkgs/notify-osd/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'notify-osd'
+
+pkgname=notify-osd
+version=0.9.34
+revision=1
+build_style=gnu-configure
+short_desc="Canonical's on-screen-display notification agent"
+maintainer="Emanuel Serpa <emanuel@openmailbox.org>"
+hostmakedepends="pkg-config  glib-devel"
```
Comment (beefcurtains) [inline srcpkgs/notify-osd/template]: Add ``dbus-glib-devel``, fixes cross compilation (and ``glib-devel`` may as well be just ``glib`` afaict).

**Comment by xtraeme** [comment]
- You are still missing `dbus-glib-devel` in `makedepends`
- Please be more descriptive in your commit logs.
- Please rebase to remove duplicated commits.

**Comment by xtraeme** [comment]
- Cross compilation still requires `dbus-glib-devel` in `hostmakedepends`.

**Pull request merged**